src/components/CreateLogo/SelectFont/index.vue

```diff
@@ -35,38 +36,57 @@
 </template>
 
 <script>
-  import storage from '../../../model/storage'
-  import textJson from '../../../JSON/text.json'
-  import cmsJson from '../../../JSON/cms.json'
+  import Logo from '../../../models/logo'
+  import textJson from '../../../common/data/fonts.json'
+  import cmsJson from '../../../common/data/messages.json'
 
   export default {
     name: 'HelloWorld',
     data: () => ({
-      ...storage.getFontData(),
       width: '33%',
       fontItems: textJson,
-      cms: cmsJson.selectFont
+      cms: cmsJson.selectFont,
+      cardGrid: Number
     }),
-    mounted() {
-        // this.a = window.localStorage.getItem('user');
+    props: {
+      currentLogoObj : {
+        type : Object
+      }
     },
     methods: {
-        changeFont(font, fontId){
-          this.selectedFont = fontId;
-          this.logoFont = font;
-          storage.saveFontData(font, fontId);
-        },
-      example(e){
-        var idIndex = this.fontItems.findIndex(item => item.id == e);
-        var font = this.fontItems[idIndex].fontaFamily;
-        var fontId = this.fontItems[idIndex].id
-        this.changeFont(font, fontId)
+      changeFont(font, fontId){
+        const logo = Logo.instance;
+        logo.font = font;
+        logo.fontId = fontId;
+        logo.save();
+      },
+      selectFont(id){
+        let array = this.fontItems
+        let slectedArray = array.filter((e) => e.id == id )
+        const font = slectedArray[0].font;
+        this.changeIcon(font, id)
+      }
+          
+    },
+    mounted() {
+      let char = this.currentLogoObj.name.length;
+      if (char > 0 && char <= 6) {
```

Comment:
move to utility method and add comments to explain whats happening

Reply:
i am handling no. of grids ..eg:- if the name is under 6 character it will be col-md-3 nad if its more than 24 char it will become col-md-12